.golangci.yml

```diff
@@ -84,7 +84,7 @@ linters-settings:
       - ST1001
 
 linters:
-  enable:
+  disable:
```

Comment:
also make sure this change isn't in the final PR

Reply:
Okay, updated. Thanks!